Objective-C/Tests/CBLTestCase.m

```diff
@@ -90,6 +91,10 @@ - (void) tearDown {
     [super tearDown];
 }
 
++ (void) tearDown {
+    [CBLLogSinks resetApiVersion];
```

Comment:
This may not work when we want to debug an issue by enable console log in the setup method. Probably move to the regular setup method in this class?

Reply:
That is true if you enable console logging at the CBLTestCase level setup method.

I moved it under -setUp and I have added one in -tearDown as well. It is needed in case there's additional logging configuration made in test classes. For now is only in the log tests classes, but it might be the case in the future. 

Tested with enabling verbose logging using new api, before reset and it work. Marked with a comment.